### PR TITLE
Change default log level to be less verbose

### PR DESCRIPTION
**Pull request by borod108**
Fix https://bugzilla.redhat.com/show_bug.cgi?id=2037421

Signed-off-by: borod108 <boris.od@gmail.com>

```release-note
Changed log level
```

### CODE REVIEW

**Comment by ksimon1** [comment]
/lgtm
I would appreciate the ability to set this from e.g. CR, but it can be done in some different PR.

**Comment by ksimon1** [comment]
@borod108 can you please remove [wip] from subject?

**Comment by borod108** [comment]
@ksimon1 I want to get the full version of the fix, this way it is very annoying, i will only ask to merge this if I will not make a full fix in time...

File: controllers/ssp_controller.go [hunk 1]
```diff
@@ -197,7 +197,7 @@ func (r *sspReconciler) Reconcile(ctx context.Context, req ctrl.Request) (res ct
 	if err != nil {
 		return ctrl.Result{}, err
 	}
-	sspRequest.Logger.V(1).Info("CR status updated")
+	sspRequest.Logger.Info("CR status updated")
```
Comment (akrejcir) [inline controllers/ssp_controller.go:200]: Can you keep these 2 log lines in debug, and add a third log line with info level at the end of this method, that will say something like: Reconciliation finished.

**Review by akrejcir — CHANGES_REQUESTED** [review]
Looks good, just one minor comment.

File: controllers/ssp_controller.go (hunk 1 above)
Comment (borod108) [reply controllers/ssp_controller.go:200]: actually this is done on purpose to give more information (if the log line it is there we have more info on what failed)

Comment (akrejcir) [reply controllers/ssp_controller.go:200]: ok

**Pull request merged**